libs/labelbox/tests/integration/test_delegated_access.py

```diff
@@ -77,3 +78,52 @@ def test_no_default_integration(client):
     ds = client.create_dataset(name="new_ds")
     assert ds.iam_integration() is None
     ds.delete()
+
+
+@pytest.mark.skip(
+    reason=
+    "Google credentials are being updated for this test, disabling till it's all sorted out"
+)
+@pytest.mark.skipif(not os.environ.get("DA_GCP_LABELBOX_API_KEY"),
+                    reason="DA_GCP_LABELBOX_API_KEY not found")
+def test_integration_change():
+    """
+    This test is based on test_non_default_integration() and assumes the following:
+    
+    1. aws delegated access is configured to work with lbox-test-bucket
+    2. an integration called aws is available to the org
+
+    Currently tests against:
+    Org ID: cl26d06tk0gch10901m7jeg9v
+    Email: jtso+aws_sdk_tests@labelbox.com
+    """
+    client = Client(api_key=os.environ.get("DA_GCP_LABELBOX_API_KEY"))
+    integrations = client.get_organization().get_iam_integrations()
+
+    # Prepare dataset with an existing integration
+    integration = [
+        integration for integration 
+        in integrations 
+        if 'aws-da-test-bucket' in integration.name][0]
+
+    ds = client.create_dataset(iam_integration=integration, name=f"integration_change-{uuid.uuid4()}")
+
+    # Test unset integration
+    ds.set_iam_integration()
```

Comment:
Split this test. it's testing 3 different codepaths (set with id, set with object, unset) in one test that could easily be separated.

Reply:
Done